dev-util/spirv-tools/spirv-tools-2021.1.ebuild

```
@@ -34,7 +34,7 @@ BDEPEND="${PYTHON_DEPS}
 
 multilib_src_configure() {
 	local mycmakeargs=(
-		"-DSPIRV-Headers_SOURCE_DIR=/usr/"
+		"-DSPIRV-Headers_SOURCE_DIR=${EPREFIX}/usr/"
```

This should ideally be ESYSROOT because that's [where](https://dev.gentoo.org/~ulm/pms/head/pms.html#x1-72002r2) DEPEND-class dependencies are satisfied:
```suggestion
		"-DSPIRV-Headers_SOURCE_DIR=${ESYSROOT}/usr/"
```

---

Thanks for pointing out. I had no experience on cross-compiling before, but after briefly reading some documents I get a bit clearer. I think you're right about using ESYSROOT instead of EPREFIX.

---

Should I change it on my branch or you just push the correct one to the master in a new commit?

---

Preferable to just force-push to this branch with fixes, then could use your commit as-is.

---

Force pushing done.